src/components/rounds-list/rounds-list.tsx

```diff
@@ -0,0 +1,24 @@
+import { Round } from "@/app/events/[event]/participants/page";
```

Comment:
If this type will be shared, we should move it to a common/shared folder.

Reply:
Done